packages/orbit-components/src/Accordion/AccordionSection/index.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import * as React from "react";
+
+import { useAccordion } from "../AccordionContext";
+import randomID from "../../utils/randomID";
+import useBoundingRect from "../../hooks/useBoundingRect";
+import Slide from "../../utils/Slide";
+import AccordionWrapper from "../components/AccordionWrapper";
+import SectionHeader from "./components/SectionHeader";
+import SectionFooter from "./components/SectionFooter";
+import SectionContent from "./components/SectionContent";
+
+import type { Props } from "./index";
+
+const AccordionSection = ({
+  children,
+  header,
+  footer,
+  actions,
+  dataTest,
+  expandable = true,
+}: Props) => {
+  const { expanded, onExpand } = useAccordion();
+
+  const slideID = React.useMemo(() => randomID("slideID"), []);
+  const labelID = React.useMemo(() => randomID("labelID"), []);
+
+  const [{ height }, ref] = useBoundingRect({ height: expanded ? null : 0 });
+
+  return (
+    <AccordionWrapper dataTest={dataTest}>
+      {header && (
+        <SectionHeader
+          actions={actions}
+          expanded={Boolean(expanded)}
+          onExpand={onExpand}
+          expandable={expandable}
+        >
+          {header}
+        </SectionHeader>
+      )}
+
+      <Slide maxHeight={height} expanded={expanded} id={slideID} ariaLabelledBy={labelID}>
```

Comment:
The value of [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute) should be same as the `id` attribute of an element that contains its label. Or something like that, I'm not familiar with this part of a11y, but the internet is.